pkg/upgrade/container/container.go

```diff
@@ -46,7 +46,7 @@ func WithImageTag(tag string) Option {
 	return func(container *corev1.Container) {
 		image := container.Image
 		if p := strings.Split(image, `:`); len(p) > 1 {
-			image = strings.Join(p[0:len(p)-1], `:`)
+			return
```

Comment:
This should cause some of the test runs for `WithImageTag` to fail. Will merge anyway and fix later.